### DISCUSSION TITLE
I've completed Github Intro complete But not updating

### DISCUSSION THREAD

**Discussion by imsurajj** [community/community#141787] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've completed each step to complete the module of **intro to github**  

The readme file shows the finished part but this process is not showing as finished on the GitHub education dashboard.

How can I resolve this issue

**Comment by imsurajj** [comment]
![Screenshot 2024-10-17 184538](https://github.com/user-attachments/assets/ef468420-5312-42f2-a0df-5a8a0790c43e)


This status is not updating

**Reply by umer-tariq77400** [reply]
Same issue bro. Tell me how you resolved it?
Please

**Comment by MorelliniMatteo** [comment, marked as answer]
It sounds like you've completed all the steps in the Introduction to GitHub module, but it's not showing as finished on the GitHub Education dashboard. Here are some steps to troubleshoot and resolve this issue:

1. Verify Completion on GitHub:
Double-check that you’ve committed and pushed all changes to the repository where the module tasks are tracked (e.g., completing the README file).
Ensure that all steps required by the module are marked as done (you may need to mark checkboxes or click a "complete" button in some modules).
2. Check Your GitHub Education Dashboard:
Visit your GitHub Education dashboard and check if there’s any pending task or action related to the module that you missed.
Sometimes the system may take time to update, so give it a few minutes after completing the last task.
3. Clear Browser Cache or Try Different Browser:
Occasionally, the dashboard might not refresh properly due to cached data. Try clearing your browser’s cache or opening the dashboard in a different browser to see if the progress updates.
4. Revisit the Module:
Go back to the module and check for any steps that might have been skipped or overlooked. Some modules have specific tasks that require manual confirmation or clicking on certain items before they are marked as complete.
5. Check for Known Issues or Updates:
Visit the GitHub Education community forums or GitHub Support to see if there are any known issues with module completion or if others are experiencing similar problems.
6. Contact GitHub Education Support:
If none of the above steps help, you can reach out to GitHub Education Support. Provide them with details about the specific module and the steps you’ve completed, and they can investigate why it’s not showing as finished.
By following these steps, you should be able to resolve the issue and have your module marked as complete on the dashboard.

**Reply by imsurajj** [reply]
I recheck this again and i found one part skipped, can you tell me how can I resolve that

**On open a pull request is not done**

![Screenshot 2024-10-17 184844](https://github.com/user-attachments/assets/71a5820f-7810-4448-9418-a9cfde795b61)

Can you solve it...

**Reply by MorelliniMatteo** [reply]
To resolve the issue with the "On open a pull request" step not being completed, follow these steps:

Revisit the Pull Request:

Go to your GitHub repository and navigate to the "Pull Requests" tab.
Check if there’s an open pull request for the module. If not, create one by clicking on "New Pull Request."
Create or Update the Pull Request:

Make sure your pull request includes all the necessary changes as per the module instructions.
If you already have a pull request, ensure it reflects the latest changes from your branch.
Merge the Pull Request:

After ensuring that all changes are correct, you can merge the pull request by clicking "Merge pull request."
Follow any additional instructions provided in the module for merging.
Check the Dashboard Again:

After merging, wait a few minutes and then check your GitHub Education dashboard to see if the module is now marked as complete.
By completing these steps, the "On open a pull request" task should be marked as done. If you continue to have issues, consider contacting GitHub Education Support for further assistance.

**Reply by MorelliniMatteo** [reply]
could you please make my first answer as answered ?
